resources/tickets.py

```diff
@@ -92,3 +92,19 @@ def post(self):
 
         ticket.save_to_db()
         return ticket.json(), 201
+
+    @pm_level_required
+    def delete(self):
```

Comment:
hmmm, I think I like this. I probably would've suggested to create a new endpoint called batch-delete or something, but this works too.